tools/firmware-utils/src/mkdlinkfw.c

```diff
@@ -114,6 +115,7 @@ static void usage(int status)
 		"  -f              set family member id (hexval prefixed with 0x)\n"
 		"  -F <file>       read image and convert it to FACTORY\n"
 		"  -k <file>       read kernel image from the file <file>\n"
+		"  -M <AUH MAGIC>  override standard AUH MAGIC string\n"
```

Comment:
you added an option to the program, but not using it?

remember the magic number is for the loader to read, not for JBoot

Reply:
nevermind I see the commit message now

Reply:
Btw, you are right @mpratt14 !
This commit is not strictly needed to this PR.

Of course, it's a nice addition.